src/swap/db.rs

```diff
@@ -139,61 +137,6 @@ impl Database {
     }
 }
 
-/// Active takers related functions
-impl Database {
```

Comment:
Are we sure we can just delete this functionality? I think nectar had a requirement of not doing more than one trade with a single person at the same time.

Reply:
Im not sure to be honest. Why cant nectar do more than one trade at a time?

Reply:
I think the reasoning was to minimize risk of trading with a bad counterparty. @d4nte do you mind chimming in here?

Reply:
Done here: https://github.com/coblox/nectar/pull/112#discussion_r480506366

Reply:
re added these checks